examples/README.md

```diff
@@ -0,0 +1,30 @@
+# Libfabric Examples
+
+The examples in this directory provide standalone libfabric examples for
+developers as starting points for writing OFI programs.  Each example includes
+detailed comments that explain every OFI call and its purpose.
+
+A single Makefile is provided for simplicity. Set the environment variable
+`LIBFABRIC_PREFIX` to specify where to find the libfabric library and headers.
+
+## Examples
+
+### fi_example_rdm (rdm.c)
+
+This example uses `FI_EP_RDM` (reliable, unconnected) with a simple server and
+client to execute a one-sided send and receive.
```

Comment:
use `one-directional` or `one-way`.  `one-sided send/receive` is a confusing term.